runtime/doc/builtin.txt

```diff
@@ -1169,6 +1172,13 @@ balloon_split({msg})					*balloon_split()*
 <		{only available when compiled with the |+balloon_eval_term|
 		feature}
 
+bindtextdomain({package} [, {path}])			*bindtextdomain()*
+		Bind a specific {package} to a {path} so that the
+		|gettext()| function can be used to get language-specific
+		translations for a package.
```

Comment:
```suggestion
		translations for a package.  {path} should typically use the lang directory name see |package-create|
```

Reply:
Added that. Also made the {path} parameter obligatory. It's nonsense to fall back to the $VIMRUNTIME path. If you want to get a string from there, just use `gettext()` without the package...